pkg/auth/csp/grpc.go

```diff
@@ -144,7 +145,7 @@ func GetAuthOptsOrExit() grpc.CallOption {
 	var err error
 	authOpts, err = WithCredentialDiscovery()
 	if err != nil {
-		log.Fatal("Not logged in. Please retry after logging in")
+		log.Fatal(errors.New("not logged in. Please retry after logging in"), "")
```

Comment:
I would log the actual error along with messages, not sure why we are skipping the err